### ISSUE TITLE
Constructed transaction does not parse correctly

### ISSUE THREAD

**Issue by jdogresorg** [CounterpartyXCP/counterparty-core#1119]
It seems that we have an issue with transferring asset ownership to segwit (bech32) addresses, as the API throws an error indicating that the constructed transaction does not parse properly.

API Request:
```
{
	"method": "create_issuance",
	"params": {
		"source": "1DDEJ8E7qmeL2uvY8yaRSjrfHQb8wkrAvF",
		"asset": "RALPHBAKSHI",
                "transfer_destination": "bc1qljtsutdcc47jn4585l0xgqw4atdqkqvc92vqym",
		"quantity":0,
		"allow_unconfirmed_inputs": true
	},
	"jsonrpc": "2.0",
	"id": 0
}
```

API Response:
```
{
    "jsonrpc": "2.0",
    "id": 0,
    "error": {
        "message": "Error composing issuance transaction via API: Constructed transaction does not parse correctly: ('1DDEJ8E7qmeL2uvY8yaRSjrfHQb8wkrAvF', 'bc1qljtsutdcc47jn4585l0xgqw4atdqkqvc92vqym', b'\\x14\\x00\\x08\\x88\\xd7\\xd4V2\\xb6\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x01\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00') ≠ ('1DDEJ8E7qmeL2uvY8yaRSjrfHQb8wkrAvF', '', b'\\x14\\x00\\x08\\x88\\xd7\\xd4V2\\xb6\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x01\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\x00')",
        "code": -32001
    }
}
```

![image](https://user-images.githubusercontent.com/376028/118007030-a08e8a00-b300-11eb-8e44-b56d7d2a5ea2.png)

**Comment by adamkrellenstein** [comment]
We're still running into this, I think.

**Comment by jdogresorg** [comment]
able to reproduce the issue/error consistently on testnet via the following request.

TESTNET
https://api.counterparty.io:14000

```
{
    "method": "create_send",
    "params": {
        "source": "mvThcDEbeqog2aJ7JNj1FefUPaNdYYGqHt",
        "destination": "tb1q75dtzykyxwe80rn3mnv2hpxtl4zawuncyhk8t2",
        "asset": "BTC",
        "quantity": 1000000,
        "fee": 1000,
        "allow_unconfirmed_inputs": true
    },
    "jsonrpc": "2.0",
    "id": 0
}
```

![telegram-cloud-photo-size-4-5824833676628833057-y](https://github.com/CounterpartyXCP/counterparty-core/assets/376028/87ddf383-431c-43ca-bb9b-c3c5dd73893f)

**Comment by adamkrellenstein** [comment]
It seems like dispensers have suddenly stopped working, and it may have something to do with this...

**Comment by adamkrellenstein** [comment]
Fixed with #1647 

```
> curl -X POST http://api3.counterparty.io:14000/api/ --user rpc:rpc -H 'Content-Type: application/json; charset=UTF-8' -H 'Accept: application/json, text/javascript' --data-binary '{
                  "method": "create_send",
                  "params": {
                      "source": "mvThcDEbeqog2aJ7JNj1FefUPaNdYYGqHt",
                      "destination": "tb1q75dtzykyxwe80rn3mnv2hpxtl4zawuncyhk8t2",
                      "asset": "BTC",
                      "quantity": 1000000,
                      "fee": 1000,
                      "allow_unconfirmed_inputs": true
                  },
                  "jsonrpc": "2.0",
                  "id": 0
              }'
{"result": "0100000001680c6f5161dab4f86142eef5fe7434e9fa28744de1e3d04a55a18d234bce5536010000001976a914a3ec60fb522fdf62c90eec1981577813d8f8a58a88acffffffff0240420f0000000000160014f51ab112c433b2778e71dcd8ab84cbfd45d7727867a64a1c000000001976a914a3ec60fb522fdf62c90eec1981577813d8f8a58a88ac00000000", "id": 0, "jsonrpc": "2.0"}
>
```
